internal/cli/serverless/sqluser/create.go

```diff
@@ -0,0 +1,260 @@
+
+				// interactive mode
+				project, err := cloud.GetSelectedProject(h.QueryPageSize, d)
+				if err != nil {
+					return err
+				}
+				projectID := project.ID
+
+				cluster, err := cloud.GetSelectedCluster(projectID, h.QueryPageSize, d)
+				if err != nil {
+					return err
+				}
+				clusterID = cluster.ID
+
+				userRole, err = cloud.GetSelectedBuildinRole()
+				if err != nil {
+					return err
+				}
+
+				// variables for input
+				fmt.Fprintln(h.IOStreams.Out, color.BlueString("Please input the following options"))
+
+				p := tea.NewProgram(initialCreateInputModel())
+				inputModel, err := p.Run()
+				if err != nil {
+					return errors.Trace(err)
+				}
+				if inputModel.(ui.TextInputModel).Interrupted {
+					return util.InterruptError
+				}
+
+				userName = inputModel.(ui.TextInputModel).Inputs[createSQLUserField[flag.UserName]].Value()
+				password = inputModel.(ui.TextInputModel).Inputs[createSQLUserField[flag.Password]].Value()
+
+			} else {
+				// non-interactive mode doesn't need projectID
+				cID, err := cmd.Flags().GetString(flag.ClusterID)
+				if err != nil {
+					return errors.Trace(err)
+				}
+				clusterID = cID
+
+				uName, err := cmd.Flags().GetString(flag.UserName)
+				if err != nil {
+					return errors.Trace(err)
+				}
+				userName = uName
+
+				pw, err := cmd.Flags().GetString(flag.Password)
+				if err != nil {
+					return errors.Trace(err)
+				}
+				password = pw
+
+				uRole, err := cmd.Flags().GetString(flag.UserRole)
+				if err != nil {
+					return errors.Trace(err)
+				}
+				userRole = uRole
+			}
+
+			// generate the built-in role
+			builtinRole := GetBuiltInRole(userRole)
+
+			params := iamApi.NewPostV1beta1ClustersClusterIDSQLUsersParams().
+				WithClusterID(clusterID).
+				WithSQLUser(
+					&iamModel.APICreateSQLUserReq{
+						AuthMethod:  util.MYSQLNATIVEPASSWORD,
+						UserName:    userName,
+						Password:    password,
+						BuiltinRole: builtinRole,
+					},
+				)
+
+			_, err = d.CreateSQLUser(params)
+			if err != nil {
+				return errors.Trace(err)
+			}
+
+			_, err = fmt.Fprintln(h.IOStreams.Out, color.GreenString("SQL user %s is created", userName))
+			if err != nil {
+				return err
+			}
+			return nil
+
+		},
+	}
+
+	CreateCmd.Flags().StringP(flag.Output, flag.OutputShort, output.HumanFormat, flag.OutputHelp)
+	CreateCmd.Flags().StringP(flag.ClusterID, flag.ClusterIDShort, "", "The ID of the cluster.")
+	CreateCmd.Flags().StringP(flag.UserName, "", "", "The name of the SQL user.")
+	CreateCmd.Flags().StringP(flag.Password, "", "", "The password of the SQL user.")
+	CreateCmd.Flags().StringP(flag.UserRole, "", "", "The built-in role of the SQL user.")
+
+	return CreateCmd
+}
+
+func GetBuiltInRole(userRole string) string {
+	role := strings.ToLower(userRole)
+	switch role {
+	case util.ADMIN:
+		role = util.ADMIN_ROLE
+	case util.READWRITE:
+		role = util.READWRITE_ROLE
+	case util.READONLY:
+		role = util.READONLY_ROLE
+	default:
+		role = userRole
```

Comment:
if the input is not the three, won't the API return error? it's not a valid builtin role right？